### PR TITLE
Adding user agent

### PR DESCRIPTION
**Pull request by shantanuFyndGit**

### CODE REVIEW

File: .DS_Store
Comment (maulik-gofynd) [inline .DS_Store:1]: - remove this file 
- update `.gitignore` for the same

File: src/.DS_Store
Comment (maulik-gofynd) [inline src/.DS_Store:1]: - remove this file
- update .gitignore for the same

**Review by maulik-gofynd — CHANGES_REQUESTED** [review]

File: src/.DS_Store
Comment (shantanuFyndGit) [reply src/.DS_Store:1]: removed

File: .DS_Store
Comment (shantanuFyndGit) [reply .DS_Store:1]: removed

**Review by maulik-gofynd — APPROVED** [review]

**Pull request merged**